adapter/integration-test/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN npm ci
 COPY ./.eslintrc.json ./
 COPY ./src ./src
 
-RUN npm run lint-ci
```

Comment:
Pls go over all changes outside of /adapter and see if they need to be reverted. Like this one ;)

Reply:
All linting should be added again and tests have been fixed